paddle/fluid/operators/controlflow/while_op.cc

```diff
@@ -87,14 +101,38 @@ class WhileOp : public framework::OperatorBase {
     bool is_test = Attr<bool>("is_test");
     auto &skip_vars = Attr<std::vector<std::string>>(kSkipEagerDeletionVars);
     VLOG(2) << GetSkipEagerDeletionVarsDebugString(skip_vars);
-
     auto ctx = executor.Prepare(*program, block->ID(), skip_vars);
     if (!is_test) {
       while (cond_data) {
         auto &current_scope = scope.NewScope();
         step_scopes->push_back(&current_scope);
+
+        const std::string suffix = "@TMP_COPY";
```

Comment:
If you put suffix as a constant, you should put at the head of this file and name it as "SUFFIX".

I remember you didn't use a variable in the past. The old code is also good to me.

Reply:
Or write "kSuffix" as we are following Google style.

Reply:
I just read more code and found you used this "suffix" below, then you cannot inplace it, you should put at the head of this file

Reply:
Thanks, done.